### ISSUE TITLE
[Need Help] 使用官方示例配置启动失败

### ISSUE THREAD

**Issue by longXboy** [bfenetworks/bfe#880]
二进制版本：1.3.0
bfe.conf使用此处配置：https://www.bfe-networks.net/zh_cn/configuration/bfe.conf/
其他子模块配置则直接使用1.30 release包中默认配置
具体报错信息如下：
```
[2021/10/31 13:43:56 CST] [INFO] [30196] (main.main:93) bfe[version:1.3.0] start
[2021/10/31 13:43:56 CST] [EROR] [30196] (github.com/bfenetworks/bfe/bfe_server.StartUp:45) StartUp(): InitHttps():in TlsRuleConfLoad() :BfeTlsRuleConfCheck(): example_product wrong rule invalid proto (h2;rate=100;isw=65535;mcs=200;level=0) in NextProtos: proto format invalid: invalid proto params: rate=100;isw=65535;mcs=200;level=0
[2021/10/31 13:43:56 CST] [EROR] [30196] (main.main:116) main(): bfe_server.StartUp(): in TlsRuleConfLoad() :BfeTlsRuleConfCheck(): example_product wrong rule invalid proto (h2;rate=100;isw=65535;mcs=200;level=0) in NextProtos: proto format invalid: invalid proto params: rate=100;isw=65535;mcs=200;level=0
```

**Comment by liuximu** [comment]
yes, we find go has change implement of  url.ParseQuery (https://github.com/golang/go/commit/e6dda19888180c5159460486d30c0412e4980748)。 we will fixed it in version 1.3.1.

**Comment by longXboy** [comment]
> yes, we find go has change implement of url.ParseQuery ([golang/go@e6dda19](https://github.com/golang/go/commit/e6dda19888180c5159460486d30c0412e4980748)。) we will fixed it in version 1.3.1.
got it , thx !

**Comment by iyangsj** [comment]
Fixed by #883
